npm/debugTools/dataplane/parser.go

```diff
@@ -0,0 +1,338 @@
+package dataplane
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os/exec"
+
+	"github.com/Azure/azure-container-networking/npm/util"
+)
+
+// Parser struct
+type Parser struct{}
```

Comment:
At first, my thoughts were making all parsing functions can only be call from a parser struct would be more organized and easier to understand but now I think you and Evan are right, I will make the changes.

Reply:
After reading some more codes, it seems there is advantage to use parser struct. 
It explicitly lets us know it is related to parsing logics while the methods already have "parse-" prefix.

I am not sure which one (with or without parser struct) is better  while dropping parser struct is more natural.
It would be nice to drop all parse- prefix in all methods in parser.go if we stick to use parser struct.

What do you think?

Reply:
drop the struct, put in own `parse` package. then the usage will be `parse.IptablesObject`, pure functions without needing to hold an object ref for access